### PR TITLE
IV Drip Insertion Rate Finetuning

### PR DESCRIPTION
**Pull request by RandomGamer123**
## About The Pull Request
IV drips can now have their injection rate adjusted via a TGUI.
The old left-click functionality has been moved to right-click, while alt-click (modelled after how atmos pumps work) immediately sets the injection rate to maximum (5 units / rough metabolisation cycle (more precisely 2 seconds)) which is consistent with the old "normal" rate. 
Blood packs still have their speed doubled as they used to.

Idea from Pentation#6653 on Discord in #general 

## Why It's Good For The Game
Makes syriniver and saline-glucose solution (and IVs in general) more attractive to use by removing the need for dilution.
Opens up possibilities for new medicines with more finnicky overdoses but higher healing rates by allowing for medical to manage dose rates better.

## UI
Screenshot

## Changelog

:cl:
expansion: IV Drips can now have their insertion rates finetuned to anything between 0.1-5u/cycle.
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
a new tgui menu via web edits, this has gotta be good

File: tgui/packages/tgui/interfaces/IVDrip.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    injectOnly,
+    maxInjectRate,
+    minInjectRate,
+    mode,
+    connected,
+    beakerAttached,
+    useInternalStorage,
+  } = data;
+  return (
+    <Window resizable>
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/IVDrip.js:18]: wtf man no dimensions

**Comment by TheBonded** [comment]
Wasn't the metabolism cycle halved to roughly 1 second? Pretty sure the wiki wasn't updated but I know I've heard people talk about that

**Comment by ExcessiveUseOfCobblestone** [comment]
Did I miss it or is there no rounding here?  can I do .XX and further?

as long as the min and max are the current IV options I’m ok with this, if it’s lower then Im not sold yet on bring an out of the box option

**Comment by RandomGamer123** [comment]
Oh yeah I forgot to add a rounding. Currently min is 0.1 which is what i intended values to be rounded to (ie 0.1 accuracy) compared to the old "slow" iv option of 1, maximum is 5 which is consistent with the old "normal" iv option.

**Review by stylemistake — APPROVED** [review]

**Pull request merged**